config.toml.example

```diff
@@ -23,13 +23,6 @@ PREFERRED_URL_SCHEME = "http"
 TEMPLATES_AUTO_RELOAD = ""
 MAX_COOKIE_SIZE = ""
 
```

Comment:
Please also add the `SQLALCHEMY_DATABASE_URI` variable with an example value of, say, `sqlite:////tmp/w2fm.db`, so that people have an example of how to configure the database.
I would put it close to the top because it's pretty important.